packages/perseus/src/server-item-renderer.tsx

```diff
@@ -298,13 +300,26 @@ export class ServerItemRenderer
         return this.props.item.hints.length;
     }
 
+    /**
+     * Returns an object of the widget `.getUserInput()` results
+     */
+    getUserInputMap(): UserInputMap {
```

Comment:
Nit: can we just call this `getUserInput()`? We do type the return value and adding the return type as part of the function name just feels like 90's.